internal/core/ports/feeder.go

```diff
@@ -0,0 +1,28 @@
+package ports
+
+import "context"
+
+type PriceFeeder interface {
+	// Start starts the price feeder service which is periodically fetching
+	//and updating market prices from external price provide
+	Start(ctx context.Context, markets []Market) (chan PriceFeedChan, error)
+	// Stop stops the price feeder service
+	Stop(ctx context.Context)
+	// StartFeed starts updating specific price feed for the given feedID
+	StartFeed(ctx context.Context, feedID string) error
+	// StopFeed stops updating specific price feed for the given feedID
+	StopFeed(ctx context.Context, feedID string) error
+	// AddPriceFeed adds a new price feed
+	//it is necessary to start the feed after adding it in order to start updating prices
+	AddPriceFeed(ctx context.Context, req AddPriceFeedReq) (string, error)
+	// UpdatePriceFeed updates an existing price feed
+	UpdatePriceFeed(ctx context.Context, req UpdatePriceFeedReq) error
+	// RemovePriceFeed removes an existing price feed
+	RemovePriceFeed(ctx context.Context, feedID string) error
+	// GetPriceFeed returns the price feed for the given market
+	GetPriceFeed(ctx context.Context, baseAsset string, quoteAsset string) (*PriceFeed, error)
```

Comment:
Do we really need this API if we add `ListPriceFeeds()`? If I'm not wrong this is currently used when starting the price feeds. If we still need this we can call it:
```suggestion
	GetPriceFeedForMarket(ctx context.Context, baseAsset string, quoteAsset string) (*PriceFeed, error)
```

Reply:
Done